.gitignore

```diff
@@ -5,3 +5,4 @@ config/prod.php
 dev_db.sqlite3
 build/
 .sass-cache
+assets/vid/IIB 10-06-16 Segment 5A.mp4
```

Comment:
Originally it was in the repo but then I realized it was too large to upload to Github. So I deleted it from the repo and had Mikel upload it to Vimeo. I tried to push to Github again and it still gave me errors. Apparently it was still in the history of commits and I had all kinds of trouble trying to remove it from the history. After surfing through forums I finally found the right command and removed it. I'll take that out of the .gitignore.